### PR TITLE
add redis4.0 moduleExecute

### PR DESCRIPTION
**Pull request by jingbanghu**
if there is something wrong,please tell me

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -168,29 +168,29 @@
...
-				    <execution>
-				      <id>bundle-manifest</id>
-				      <phase>process-classes</phase>
-				      <goals>    
-				        <goal>manifest</goal>
-				      </goals>   
-				    </execution>
-				</executions>		
-			</plugin>
+<!-- 			<plugin> -->
```
Comment (marcosnils) [inline pom.xml]: Hmm is there a reason why this is commented?

File: src/main/java/redis/clients/jedis/BinaryClient.java [hunk 2]
```diff
@@ -1,1303 +1,1312 @@
-package redis.clients.jedis;
```
Comment (marcosnils) [inline src/main/java/redis/clients/jedis/BinaryClient.java]: This whole class changed?

File: src/main/java/redis/clients/jedis/Client.java [hunk 3]
```diff
@@ -1,1220 +1,1230 @@
-package redis.clients.jedis;
```
Comment (marcosnils) [inline src/main/java/redis/clients/jedis/Client.java]: This whole class changed?

**Comment by jingbanghu** [comment]
i will change it later

**Pull request closed without merging**